### DISCUSSION TITLE
Multiple time zones?

### DISCUSSION THREAD

**Discussion by acr1161** [simpleavr/s14clock#14] [Q&A] [answered]
Just received and assembled clock this weekend. Still feeling my way around with interface. Is there a way to display more than one time zone?

**Comment by simpleavr** [comment]
Not at the moment.
This sound like a good feature though.
Perhaps I can add a token to acheive this.
Something like ~+6, ~-2 to indicate GMT offset number of hours (and enter at start of contents). This will be relatively easy to implement.
Or you have other ideas.
Look for notification on this thread.

**Comment by simpleavr** [comment, marked as answer]
I had add new tokens "\~+dd" and "\~-dd" so that we can adjust dd hours to the specific content string.

**2013-07-25 Implement new content tokens '\~+dd' and '\~-dd' to adjust current time by adding / subtracting hours, useful for multi-city display.**

Please note that the hours offset is relative to your current setting (**NOT** relative to GMT). If you are in New York and your want to show L.A. time in one of the content slots, use "\~-3Los Angeles", "%X".

Latest firmware_202.bin contain this change.
Please give feedback or "OK" solution if you try this. Thanks.
